drivers/smbus/smbus_stm32.c

```diff
@@ -0,0 +1,318 @@
+/*
+ * Copyright (c) 2023 SILA Embedded Solutions GmbH
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/kernel.h>
+#include <zephyr/device.h>
+#include <zephyr/drivers/i2c.h>
+#include <zephyr/drivers/i2c/stm32.h>
+#include <zephyr/drivers/pinctrl.h>
+#include <zephyr/drivers/smbus.h>
+#include <zephyr/logging/log.h>
+#include <zephyr/sys/byteorder.h>
+#include <soc.h>
+
+#include "smbus_utils.h"
```

Comment:
Do you mean I should add them already in smbus_utils.h? I don't think so, because I would try to include as little as possible, and smbus_utils.h doesn't reference anything in byteorder.h and soc.h

Reply:
I meant you added them already to `smbus_utils.h` :) I agree we should include as little as possible.